infrastructure/README.md

```diff
@@ -4,10 +4,10 @@ This folder contains all the files for provisioning infrastructure in Azure.
 
 The following instructions are divided in:
 
-- [Provisioning a new environment](provisioning-in-a-new-environment)
-- [Provisioning infrastructure in an existing environment](provisioning-infrastructure-in-an-existing-environment)
-- [Connecting to a Kubernetes cluster](connecting-to-a-kubernetes-cluster) using `kubectl`
-- [Installing Istio](installing-istio)
+- [Provisioning a new environment](#provisioning-in-a-new-environment)
```

Comment:
need to drop the `-in` so the link is `#provisioning-a-new-environment` to match the `h2` below